README.md

```diff
@@ -74,6 +74,29 @@ received:
 If the Go standard library had defined an interface for logging, this project
 probably would not be needed.  Alas, here we are.
 
+When the Go developers started developing such an interface with
+[slog](https://github.com/golang/go/issues/56345), they adopted some of the
+logr design but also left out some parts and changed others:
+
+| Feature | logr | slog |
+|---------|------|------|
+| High-level API | `Logger` (passed by value) | `Logger` (passed by [pointer](https://github.com/golang/go/issues/59126)) |
+| Low-level API | `LogSink` | `Handler` |
+| Stack unwinding | done by `LogSink` | done by `Logger` |
+| Skipping helper functions | `WithCallDepth`, `WithCallStackHelper` | [not supported by Logger](https://github.com/golang/go/issues/59145) |
+| Generating a value for logging on demand | `Marshaler` | `LogValuer` |
+| Log levels | >= 0, higher meaning "less important" | positive and negative, with 0 for "info" and higher meaning "more important" |
```

Comment:
s/less important/more verbose

The `log.V(n)` notation is not really saying "this logline has level n" but "the verbosity of the logger must be at least n"

Reply:
`>= 0, higher meaning "more verbose"` sounds backwards to me when comparing a log call like `logger.V(10).Info("hello")` with `logger.V(5).Info("world")`. The first call is *less* verbose because it's less likely to be emitted.

"Higher = more verbose" is true when configuring a logger initially, but this here is a comparison of log calls.

Reply:
This is a super minor point.  We can drop it.  I prefer to say something like "V() indicates the minimum verbosity level required to emit this log; a higher V level requires that the app be configured to be more verbose" when I talk about this pattern.